Expand Difficulty Reduction mod articles + Add "Mods (lazer)" on the Main page (#12600)
ppy/osu-wiki: wiki/Gameplay/Game_modifier/No_Fail_(lazer)/en.md

*For the osu!stable version of this article, see: [No Fail (mod)](/wiki/Gameplay/Game_modifier/No_Fail)*\
*For the full list of all [lazer](/wiki/Client/Release_stream/Lazer) mods, see: [Game modifier (lazer)](/wiki/Gameplay/Game_modifier_(lazer))*

> The **No Fail** mod prevents the player from failing on any [beatmap](/wiki/Beatmap), which means that although the [health bar](/wiki/Client/Interface/Health_bar) reaches the bottom, you won't be able to fail, even if you didn't score any points. This mod has the same effect across all [game modes](/wiki/Game_mode).

Reviewer: ```suggestion
The **No Fail** mod prevents the player from failing on any [beatmap](/wiki/Beatmap), even if the [health bar](/wiki/Client/Interface/Health_bar) reaches zero or no points were scored. This mod has the same effect across all [game modes](/wiki/Game_mode).
```

> **This mod can't be customised through Customisation.**

Reviewer: for wiki articles, contractions should be expanded

```suggestion
**This mod cannot be customised.**
```
